### ISSUE TITLE
Roundcube installed using links not named roundcube* do not work

### ISSUE THREAD

**Issue by james-fitzrandolph** [roundcube/roundcubemail#6053]
Since 1.1.1 I have installed roundcube in directories named /var/www/roundcube-x.y.z, and had a link named roundcube which pointed at the production version.  That allows me to install the new version and test until I am happy, then change the link.  (Yes, I know, I have to work db issues.  Small price).  This has worked until 1.3.2.  It appears that 1.3.2 requires that the link be named roundcubemail, not roundcube.

If the link is named roundcube, I get a main screen menu that looks like this:
![rc-problem-1](https://user-images.githubusercontent.com/33819225/33043317-49443a56-ce02-11e7-8640-b6542e77eeae.png)
I also get the attachment menu (on the Compose-new-email screen) that looks like this: 
![rc-problem-2](https://user-images.githubusercontent.com/33819225/33043422-a70dffb4-ce02-11e7-9988-a88b15ef4385.png)

I have tested my 'fix' by creating both links and going back and forth between them.  Whenever the 'roundcube' link is used, the display is improperly drawn, whenever the 'roundcubemail' link is used, it is correct.  Using the full directory name (roundcubemail-1.3.2) also works.

My question is:  is this intentional?  I didn't see anything about this in the release notes, but am quite willing to believe that I missed it.  I can work around the issue (easy enough to create another link), but if this not on purpose, has something else been broken by accident?

**Comment by alecpl** [comment]
I don't think it's Roundcube issue. Do you see any errors in browser console?

**Comment by james-fitzrandolph** [comment]
I'm willing to believe you, mostly because I couldn't think of any reason for things to misbehave to begin with.  So, I looked (again) in the http error logs, the roundcube logs, and didn't see anything there.  And I did this with starting/stopping my browser (chrome) at random points in the process.  I brought up the chrome dev page to look at the console log, and got no errors there.  And ... the problem has gone away since then.  I can't get it to come back.  There have been no other changes to my system to cause this.  I'm totally confused, but willing to let it go because I can't think of anyplace else to go.

Thanks for the help.  Close the ticket.
